hack/update-deps.sh

```diff
@@ -1,7 +1,24 @@
-#!/bin/bash
+#!/bin/bash -e
 
-set -e
+readonly REQUIRED_GLIDE_VERSION="v0.13"
+
+function verify_glide_version() {
+	if ! command -v glide &> /dev/null; then
+		echo "[FATAL] Glide was not found in \$PATH. Please install version ${REQUIRED_GLIDE_VERSION} or newer."
+		exit 1
+	fi
+
+	local glide_version
+	glide_version=($(glide --version))
+	if [[ "${glide_version[2]}" < "${REQUIRED_GLIDE_VERSION}" ]]; then
```

Comment:
REQUIRED_GLIDE_VERSION has a `v` as first character. My glide has not. This feels broken.

Reply:
That should be easily fixable...

Reply:
It wasn't not that easy, but it's doable 😉